src/Mod/Part/App/TopoShapeWirePy.xml

```diff
@@ -14,22 +14,25 @@
       <Author Licence="LGPL" Name="Juergen Riegel" EMail="Juergen.Riegel@web.de" />
       <UserDocu>TopoShapeWire is the OpenCasCade topological wire wrapper</UserDocu>
     </Documentation>
-	  <Methode Name="makeOffset" Const="true">
-		  <Documentation>
-              <UserDocu>Offset the shape by a given amount. DEPRECATED - use makeOffset2D instead.</UserDocu>
-		  </Documentation>
-	  </Methode>
-		<Methode Name="add">
-			<Documentation>
-				<UserDocu>Add an edge to the wire</UserDocu>
-			</Documentation>
-		</Methode>
-		<Methode Name="fixWire">
-			<Documentation>
-				<UserDocu>Fix wire</UserDocu>
-			</Documentation>
-		</Methode>
-		<Methode Name="makeHomogenousWires" Const="true">
+        <Methode Name="makeOffset" Const="true">
+            <Documentation>
+                <UserDocu>Offset the shape by a given amount. DEPRECATED - use makeOffset2D instead.</UserDocu>
+            </Documentation>
+        </Methode>
+        <Methode Name="add">
+            <Documentation>
+                <UserDocu>Add an edge to the wire</UserDocu>
+            </Documentation>
+        </Methode>
+        <Methode Name="fixWire">
+            <Documentation>
+                <UserDocu>Fix wire
+A face and a tolerance can optionally be supplied to the algorithm :

```

Comment:
shouldn't it be `algorithm:` ? (ie no whitespace before colon)

Reply:
Maybe ... I don't really know.
Should be fixed anyway.
Thanks.